statusboard/src/components/PageContents/PageContents.css

```diff
@@ -1,5 +1,5 @@
 .contents {
-  display: block;
+  flex: 1;
   padding: 3rem 5vw 0 5vw;
```

Comment:
I suggest adding padding to the bottom. Right now the button on the bottom goes right up against the footer.

Reply:
It's been added